website/content/docs/install-boundary/deploy.mdx

```diff
@@ -189,4 +203,13 @@ binary:
 </Tab>
 </Tabs>
 </Tab>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+You should install the binary on the controller and worker instances you are configuring to run in you Boundary environments. 
+
+Next, you should:
+
+- [Configure Boundary controllers](/boundary/docs/install-boundary/configure-controllers)
+- [Configure Boundary workers](/boundary/docs/install-boundary/configure-workers)
+- [Initialize Boundary](/boundary/docs/install-boundary/initialize)
+- [Install the Boundary Clients](/boundary/docs/install-boundary/install-clients)
```

Comment:
This is a great section to show the next steps. If time permits, it would be good to have the same next-steps section in the Configure Boundary controllers, Configure Boundary workers, and Initialize Boundary page. Just for consistency, but no a big deal if you don't.